pkg/utils/strings/strings.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package strings
+
+func Unique(s []string) []string {
```

Comment:
We could also use [github.com/mpvl/unique](https://github.com/mpvl/unique) but should be totally fine to just create the small function here.

Reply:
after this PR got merged, my idea is create a small PR to make use of this function on other places in the code-base where it might make sense (e.g. https://github.com/giantswarm/cluster-api-provider-openstack/blob/e4be0f25a1e8182ef41da77422b9826f2c713de1/controllers/openstackmachine_controller.go#L452-L463).

If wanted i can refactor to `mpvl/unique` as well.